com.unity.render-pipelines.high-definition/Runtime/Lighting/LightUtils.cs

```diff
@@ -194,7 +196,7 @@ public static float ConvertEvToLux(float ev, float distance)
         public static float ConvertLuminanceToEv(float luminance)
         {
             float k = ColorUtils.s_LightMeterCalibrationConstant;
-            return (float)Math.Log((luminance * 100f) / k, 2);
+            return Mathf.Log(luminance, 2) + s_evToLuminanceFactor;
```

Comment:
name of the constant is a bit weird in this context

Reply:
I made two variables to clarify things